packages/lib-sourcify/test/verification.spec.ts

```diff
@@ -341,6 +341,25 @@ describe('lib-sourcify tests', () => {
 
       expectMatch(match, 'perfect', childAddress);
     });
+    it('should verify a contract passing files with wrong newlines', async () => {
+      const contractFolderPath = path.join(
+        __dirname,
+        'sources',
+        'WrongMetadata'
+      );
+      const [deployedAddress] = await deployFromAbiAndBytecode(
+        localWeb3Provider,
+        contractFolderPath,
+        accounts[0]
+      );
+
+      const match = await checkAndVerifyDeployed(
+        contractFolderPath,
+        sourcifyChainGanache,
+        deployedAddress
+      );
+      expectMatch(match, 'perfect', deployedAddress);
+    });
```

Comment:
Are you deploying the contract compiled with "Windows EOL(endofline)" and then verifying again with "Windows EOL" sources or is the "artifact.json" compiled with "Unix EOL"?

If former this is not how it works. If latter, we should explicitly comment somewhere that this bytecode ao nd metadata hash belongs to Unix EOL formatted source code.

Also I believe "SimplyLog.json" file is not being used, we can remove it.

Finally there should be a better way to name this case. Simply "wrong" doesn't tell much. In detail what we have here is a contract compiled and deployed with "Unix EOL" sources but during verification, the source files (from Etherscan i.e. without a metadata file) are provided with "Windows EOL".